src/commands/config.command.ts

```diff
@@ -0,0 +1,63 @@
+import { Command, ensureFile, Input, prompt } from "../deps.ts";
+import dir from "https://deno.land/x/dir@1.5.1/mod.ts";
```

Comment:
suggestion: try this; also see the next comment regarding export in `deps.ts`:

```suggestion
import { Command, ensureFile, Input, prompt, dir } from "../deps.ts";
```